.env

```diff
@@ -0,0 +1 @@
+MONGO_URL=mongodb+srv://user-read-write1:sfd$42A3a@horondi-yl27g.mongodb.net/horondi-1?retryWrites=true&w=majority
```

Comment:
Is it safe to post DB credentials to publicly available repository?

Reply:
no it is not safe. it is temporary for our use